components/style/properties/gecko.mako.rs

```diff
@@ -1466,6 +1296,195 @@ fn static_assert() {
     }
 </%def>
 
+fn set_image(image: Image, mut geckoimage: &mut structs::nsStyleImage, with_url: bool, cacheable: &mut bool) {
```

Comment:
Good. Ultimately we need to make the border-image-source glue match background-image, but for now set with_url to false for border-image-source and leave a todo comment